test_download/download.py

```diff
@@ -148,7 +149,7 @@ def _get_url(url, md5=False):
 
 def _get_dependencies(url):
     version = args.salt_version.replace('.', '_')
-    os = url.split('/')[5]
+    os = url.split('/')[4]
```

Comment:
i forgot to push this change but I believe its 4 for live and 5 for staging.

maybe `if 'staging' in url:`...etc?

Reply:
Ah. Changing. Also the bash script :b:roke